platform-emf/src/main/java/com/softicar/platform/emf/attribute/AbstractEmfAttribute.java

```diff
@@ -258,6 +266,22 @@ public final AbstractEmfAttribute<R, V> setColumnHandlerFactory(Supplier<IEmfDat
 		return this;
 	}
 
+	// -------------------------------- Help Display Factory -------------------------------- //
+
+	public final AbstractEmfAttribute<R, V> setHelpDisplayFactory(Supplier<IDomElement> helpDisplayFactory) {
+
+		configuration.setHelpDisplayFactory(helpDisplayFactory);
+		return this;
+	}
+
+	public final AbstractEmfAttribute<R, V> setHelpDisplayFactory(IDisplayString helpText) {
```

Comment:
IMO, setHelpDisplayFactory(IDisplayString) should be renamed to setHelpDisplay(IDisplayString)

Reply:
indeed